packages/cosmic-swingset/lib/ag-solo/vats/lib-wallet.js

```diff
@@ -226,30 +223,22 @@ export async function makeWallet(
     return offerOk;
   }
 
-  const getLocalUnitOps = assay =>
-    Promise.all([
-      E(assay).getLabel(),
-      E(assay).getExtentOps(),
-    ]).then(([label, { name, extentOpsArgs = [] }]) =>
-      makeUnitOps(label, name, extentOpsArgs),
-    );
-
   // === API
 
-  async function addAssay(assayPetname, regKey, assay) {
-    assayPetnameToAssay.init(assayPetname, assay);
-    assayToAssayPetname.init(assay, assayPetname);
-    regKeyToAssayPetname.init(regKey, assayPetname);
-    assayPetnameToRegKey.init(assayPetname, regKey);
-    petnameToUnitOps.init(assayPetname, await getLocalUnitOps(assay));
```

Comment:
Let's keep this map but change it to `petnameToAmountMath` to store the local amountMath